app/connectors/DesConnector.scala

```diff
@@ -53,8 +53,9 @@ case object DesGetUnexpectedResponse extends DesGetResponse
 class DesConnector @Inject()(val runModeConfiguration: Configuration,
                              metrics: ApplicationMetrics,
                              http: HttpClient,
-                             auditConnector: AuditConnector)
-  extends RawResponseReads with ServicesConfig {
+                             auditConnector: AuditConnector,
+                             val servicesConfig: ServicesConfig)
+  extends RawResponseReads {
 
   protected def mode: Mode = Play.current.mode
```

Comment:
check if this is used anywhere...if not delete it

Reply:
Deleted.